src/components/Receiving/ItemDetails.js

```diff
@@ -6,62 +6,160 @@ import { get } from 'lodash';
 
 import {
   Button,
+  Checkbox,
+  KeyValue,
   Modal,
+  MultiColumnList,
   Row,
   Select,
   TextField,
-  MultiColumnList,
 } from '@folio/stripes/components';
 
+import getLocationsForSelect from '../Utils/getLocationsForSelect';
+import css from './ItemDetails.css';
+
+const ITEM_STATUS = {
+  received: 'Received',
+};
+
 class ItemDetails extends Component {
   static propTypes = {
     close: PropTypes.func.isRequired,
     itemList: PropTypes.arrayOf(PropTypes.object).isRequired,
+    locations: PropTypes.arrayOf(PropTypes.object).isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      allChecked: false,
+      checkedItems: [],
+    };
+  }
+
+  getFooter = (close) => (
+    <Row end="xs">
+      <Button
+        onClick={close}
+      >
+        <FormattedMessage id="ui-orders.receiving.cancelBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled
+      >
+        <FormattedMessage id="ui-orders.receiving.previousBtn" />
+      </Button>
+      <Button
+        buttonStyle="primary"
+        disabled={!this.state.checkedItems.length}
+      >
+        <FormattedMessage id="ui-orders.receiving.nextBtn" />
+      </Button>
+    </Row>
+  );
+
+  toggleAll = (itemList) => {
+    if (!this.state.allChecked) {
```

Comment:
I think this condition should be checked inside `this.setState()` callback, since only there you get safe current value of `allChecked`